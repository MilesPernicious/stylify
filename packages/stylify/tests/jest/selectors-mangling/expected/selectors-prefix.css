.e:hover{
	color: red
}
.f{
	color: blue
}
.b:focus{
	color: orange
}
.a a{
	color: green
}
.c{
	color: yellow
}
.d{
	font-size: 24px
}
