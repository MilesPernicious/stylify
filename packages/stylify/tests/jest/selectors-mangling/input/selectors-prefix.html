<!--
stylify-components
	button: `
		u-color:yellow
		&--big {
			u-font-size:24px
		}
	`
/stylify-components
-->
<div class="
	u-color:blue
	hover:u-color:red
	focus:{u-color:orange}
	[a]{u-color:green}
"></div>

<div class="button button--big"></div>
