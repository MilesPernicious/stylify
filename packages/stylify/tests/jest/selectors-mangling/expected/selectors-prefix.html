<!--
stylify-components
	button: `
		u-color:yellow
		&--big {
			u-font-size:24px
		}
	`
/stylify-components
-->
<div class="
	f
	e
	b
	a
"></div>

<div class="c d"></div>
